easybuild/easyconfigs/e/ESyS/ESyS-2.3.5-foss-2016b-Python-2.7.12.eb

```diff
@@ -1,8 +1,10 @@
+# @author: jgp, gppezzi (CSCS)
+
 easyblock = 'ConfigureMake'
 
 name = 'ESyS'
 version = '2.3.5'
-versionsuffix = '-Python-2.7.12'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
(in the sense that Python is already loaded by Boost, but `pyver` is not defined in this case)